server/routerlicious/packages/services-core/src/runWithRetry.ts

```diff
@@ -22,18 +23,24 @@ export async function runWithRetry<T>(
         try {
             result = await api();
             success = true;
+            logger?.info(`Succeeded in executing ${callName} with ${retryCount} retries`);
+            Lumberjack.info(`Succeeded in executing ${callName} with ${retryCount} retries`);
```

Comment:
Maybe we should only log this if `retryCount >= 1`?

Reply:
Yes please. This log will be very noisy otherwise.